### PR TITLE
Add recipes doc

### PR DESCRIPTION
**Pull request by msasikanth**

### CODE REVIEW

File: doc/recipes.md [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+@Test
+fun `the feature is overriden in this test`() {
+  Features.overrides[Feature.SecureCall] = true
+}
+```
+
+You can read more about this usage [here](https://github.com/simpledotorg/simple-android/issues/1364)
+
+
+## Dagger Qualifiers
```
Comment (Honey14) [inline doc/recipes.md:39]: We already have a well documented `010-dagger-qualifier-migration.md`. Do we need to have this here as well?

**Review by Honey14 — COMMENTED** [review]
Looks great. Left one comment. PTAL

File: doc/recipes.md (hunk 1 above)
Comment (msasikanth) [reply doc/recipes.md:39]: Good point, we don't need to capture it again here. Removing it 👍🏽

**Review by Honey14 — APPROVED** [review]
Looks great

**Pull request merged**